test/evergreen.yml

```diff
@@ -1783,6 +1792,31 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: CC="clang -fsanitize=address" CFLAGS="-g -fno-omit-frame-pointer" --enable-diagnostic --with-builtins=lz4,snappy,zlib
+      - command: shell.exec
+        params: 
+          working_dir: "wiredtiger/build_posix/test/format"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            export ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0"
+            if [ -f /usr/bin/llvm-symbolizer ]; then
+              ASAN_SYMBOLIZER_PATH="/usr/bin/llvm-symbolizer"
+            else
+              ASAN_SYMBOLIZER_PATH=`ls /usr/bin/llvm-symbolizer* | tail -1`
+            fi
+            export ASAN_SYMBOLIZER_PATH
```

Comment:
Consider setting those ASAN related variables within `test_env_vars`. There are examples in test/evergreen.yml you may want to have a check.

Reply:
Yes `test_env_vars` should do the trick for the first export. Not sure how to manage `ASAN_SYMBOLIZER_PATH` variable.